swap/types.go

```diff
@@ -36,6 +36,7 @@ type Cheque struct {
 
 // HandshakeMsg is exchanged on peer handshake
 type HandshakeMsg struct {
+	ChainID         uint64
```

Comment:
Please leave a comment here, specifying that this is the chainID of the Ethereum network

Reply:
done